### ISSUE TITLE
Download gammapy-extra on readthedocs

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#400]
The gammapy-extra repo is currently not available on readthedocs for the docs build.

See e.g. log at https://readthedocs.org/projects/gammapy/builds/3559506/

We rely on example data files from gammapy-extra to make a bunch of images, e.g. there's an image missing here: https://gammapy.readthedocs.org/en/latest/irf/psf.html

This needs to be fixed somehow ... no nice images in the docs => no users.

I've looked a bit through the RTD docs and issues and didn't find a recommendation on how to do this ... there's no shell script one can run prior to the sphinx build to set stuff up as far as I know.

I see two options:
1. Downloading the gammapy-extra repo could happen via Python code in `docs/conf.py` ... it's clear from other projects that one has full internet access during the sphinx build.
2. add a `setup.py` and pip install directly from the git repo by listing it in `docs/rtd-pip-requirements.txt`.

On travis-ci, we currently have this in https://github.com/gammapy/gammapy/blob/master/.travis.yml#L109

```
- export GAMMAPY_EXTRA=${HOME}/gammapy-extra
```

and this in https://github.com/gammapy/gammapy/blob/master/.continuous-integration/travis/setup_dependencies_common.sh#L90 :

```
if $FETCH_GAMMAPY_EXTRA
then
  git clone https://github.com/gammapy/gammapy-extra.git $HOME/gammapy-extra
fi
```

At the moment Gammapy accesses gammapy-extra files via the `gammapy.datasets.gammapy_extra` variable (see [here](https://github.com/gammapy/gammapy/blob/master/gammapy/datasets/core.py#L249)), which uses the `$GAMMAPY_EXTRA` shell variable. I'm not sure if / how this needs to be changed so that `gammapy-extra` data files can be accessed on RTD.

@bsipocz Do you have any thoughts / advice here?

**Comment by bsipocz** [comment]
@cdeil - I'm sorry, but I don't have any RTD experience. What about having it as a submodule for rtd, or just simply adding the git repo to the rtd file as you do for reproject/wcsaxes/etc? Or that doesn't work as gammapy-extra is not a python module, but data files only, so doing this is pointless? I really have no idea and just wildly guessing :)

**Comment by cdeil** [comment]
Note to self.

Recently I put this hack in `docs/conf.py` to access files from `GAMMAPY_EXTRA` form the Sphinx build:
https://github.com/gammapy/gammapy/blob/352117bc417074223010b71c0fe8ebc8743b85c3/docs/conf.py#L173

It's not a good solution .. I've asked here for advice:
http://stackoverflow.com/questions/39677049/use-environment-variable-in-python-sphinx-image-directive

**Comment by cdeil** [comment]
Fixed by @adonath in #773
